packages/terra-date-picker/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Changed
+  * DateInput split into 3 distinct inputs that mask the previous single input.
```

Comment:
Maybe use "Date input" instead of "DateInput" so consumers are not confused with terra-date-input.

Reply:
Updated : https://github.com/cerner/terra-framework/pull/1208/commits/32e8836a3daba0ce2b70ccc29b360fe187c297e8